keyserver/loader.mjs

```diff
@@ -15,7 +15,7 @@ async function resolve(specifier, context, nextResolve) {
       continue;
     }
     const path = localPackages[pkg];
-    const url = defaultResult.url.replace(path, `keyserver/dist/${pkg}`);
+    const url = defaultResult.url.replace(`comm/${path}`, `comm/keyserver/dist/${pkg}`);
```

Comment:
Is there a way to fix your scenario without requiring that the git repo is in a folder called `comm`?

Reply:
@Ashoat please check the latest commit, fixed it without `comm` :)